### ISSUE TITLE
[Version/Feature Request] Non-Performance Version

### ISSUE THREAD

**Issue by LarveyOfficial** [sp614x/optifine#6995]
Optifine is an amazing mod that has an amazing player base, I've used it for as long as I can remember.
Recently I have started to move on a bit from optifine and have started seeking some other alternatives for perfromance.

I know they aren't replacements for optifine which is why I have come up with this idea.
Optifine offers much more than just performance boosts, it offers CEM, CIT, and many other tools that developers and resource pack creators can use.

I suggest making a version of optifine, that offers those features without any of the performance additions

Why?
I see many reasons why this could be beneficial to both the user and the developers.
This would allow users to use other mods that might not be compatible with optifine, and still be able to support and use some of the features they love
This would also help the developers when it comes to newer versions, as a non-performance version could come out much quicker, and lower the amount of "When is the next Optifine update coming out" comments from users.

Of course the idea isn't perfect but I believe it would allow many users who love optifines features to still use and support optifine, even if they aren't using the performance gain that regular optifine can offer.

_And no, im not saying this should also support fabric, you as a developer should be able to choose what platform to focus on, there are definetly other options to run optifine on fabric such as optifabric._

**Comment by vinc3nz** [comment]
That's a very good idea. It would also be possible to choose in the settings screen if you wish to enable performance optimizations.

**Comment by omerien** [comment]
I agree. Many features of OptiFine aren't in the optimisation, but in the details. Would really be useful to have it. But the developer of OptiFine is alone to develop it and the dozens of bug he has to fix so often would not give him the time to do it. And he can't put it open-source because it breaks Mojang's EULA.

**Comment by kristibektashi** [comment]
Perhaps they could bring back non-Ultra versions of Optifine

**Comment by ewanhowell5195** [comment]
i have spoken to sp614x about something like this and it is not happening
